tests/test_eval_pnd.py

```diff
@@ -0,0 +1,185 @@
+import unittest
+
+from context import grama as gr
+from context import data
+import numpy as np
+from numpy import array, pi
+from grama.models import make_ishigami
+from grama.fit import ft_gp
+
+class TestEvalPND(unittest.TestCase):
```

Comment:
Requested change: Add a unittest that checks the 3d output case; this will help guarantee the interface `eval_pnd()` works for arbitrary output dimensions.